services/src/main/java/org/exoplatform/officeonline/FileLock.java

```diff
@@ -0,0 +1,80 @@
+package org.exoplatform.officeonline;
+
+/**
+ * The Class FileLock.
+ */
+public class FileLock {
+
+  /**  The fileId. */
+  private final String fileId;
```

Comment:
We don't need fileId in the lock object. Let's make its contract simple and clear - a lock information that includes external lock ID (please tell this in the javadocs) and an internal (JCR) lock token with expiration time.

Reply:
Fixed: https://github.com/exo-addons/office-online/pull/7/commits/ffe7230c217a389e8afcbcffdde341dfee13aec2